### PR TITLE
feat: explore artworks tab

### PR DESCRIPTION
**Pull request by rajsam003**
This PR resolves [GRO-1271] 

image

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[GRO-1271]: https://artsyproduct.atlassian.net/browse/GRO-1271?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Scenes/ArtQuiz/ArtQuizResults/ArtQuizResultsTabs/ArtQuizExploreArtworks.tsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        contentContainerStyle={{
+          paddingVertical: 20,
+        }}
+      >
+        <Text variant="xs" color="black60">
+          We don't have any recommendations for you at this time.
+        </Text>
+      </StickyTabPageScrollView>
+    )
+  }
```
Comment (rajsam003) [inline src/app/Scenes/ArtQuiz/ArtQuizResults/ArtQuizResultsTabs/ArtQuizExploreArtworks.tsx:89]: The reason for this is [here](https://artsy.slack.com/archives/C01ADJNCS5D/p1675697685681359?thread_ts=1675697196.533069&cid=C01ADJNCS5D)

File: src/app/Scenes/ArtQuiz/ArtQuizResults/ArtQuizResultsTabs/ArtQuizExploreArtworks.tsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    if (savedArtworks.length <= 1) return 100
+    if (savedArtworks.length <= 3) return 8
+    return 4
+  }, [savedArtworks.length])
+
+  const queryResult = useLazyLoadQuery<ArtQuizExploreArtworksQuery>(artQuizExploreArtworksQuery, {
+    limit,
+  })
+
+  const artworks = uniqBy(
```
Comment (TMcMeans) [inline src/app/Scenes/ArtQuiz/ArtQuizResults/ArtQuizResultsTabs/ArtQuizExploreArtworks.tsx:30]: **suggestion [blocking]:** 

this component looks fine, but we have a field in MP that exposes a recommended artworks field and caches it better here and we should implement: https://github.com/artsy/metaphysics/blob/858283285c1793e74345d3b19c3baba041ca862d/src/schema/v2/quiz.ts#L38

*format via [Conventional Comments](https://conventionalcomments.org/)*

**Review by TMcMeans — COMMENTED** [review]
got a couple suggestions :)

File: src/app/Scenes/ArtQuiz/ArtQuizResults/ArtQuizResultsTabs/ArtQuizExploreArtworks.tsx [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+}) => {
+  const dimensions = useScreenDimensions()
+
+  const limit = useMemo(() => {
+    if (savedArtworks.length <= 1) return 100
+    if (savedArtworks.length <= 3) return 8
+    return 4
+  }, [savedArtworks.length])
+
+  const queryResult = useLazyLoadQuery<ArtQuizExploreArtworksQuery>(artQuizExploreArtworksQuery, {
```
Comment (MounirDhahri) [inline src/app/Scenes/ArtQuiz/ArtQuizResults/ArtQuizResultsTabs/ArtQuizExploreArtworks.tsx:26]: **issue(non-blocker):** unless necessary, I would avoid this query nesting here

File: src/app/Scenes/ArtQuiz/ArtQuizResults/ArtQuizResultsTabs/ArtQuizExploreArtworks.tsx [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+      if (!artwork.layer) return []
+      return extractNodes(artwork.layer.artworksConnection)
+    }),
+    "internalID"
+  )
+
+  return (
+    <StickyTabPageScrollView
+      contentContainerStyle={{
+        paddingVertical: 20,
```
Comment (MounirDhahri) [inline src/app/Scenes/ArtQuiz/ArtQuizResults/ArtQuizResultsTabs/ArtQuizExploreArtworks.tsx:41]: **issue:** let's use space from palette mobile here

File: src/app/Scenes/ArtQuiz/ArtQuizResults/ArtQuizResultsTabs/ArtQuizExploreArtworks.tsx [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+      }}
+    >
+      <GenericGrid
+        artworks={artworks}
+        width={dimensions.width - 20}
```
Comment (MounirDhahri) [inline src/app/Scenes/ArtQuiz/ArtQuizResults/ArtQuizResultsTabs/ArtQuizExploreArtworks.tsx:46]: same comment as earlier, we better use space from palette

File: src/app/Scenes/ArtQuiz/ArtQuizResults/ArtQuizResultsTabs/ArtQuizExploreArtworks.tsx [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+
+  if (isStaging) {
+    return (
+      <StickyTabPageScrollView
+        contentContainerStyle={{
+          paddingVertical: 20,
+        }}
+      >
+        <Text variant="xs" color="black60">
+          We don't have any recommendations for you at this time.
```
Comment (MounirDhahri) [inline src/app/Scenes/ArtQuiz/ArtQuizResults/ArtQuizResultsTabs/ArtQuizExploreArtworks.tsx:84]:
```suggestion
          We don't have any recommendations on staging
```

File: src/app/Scenes/ArtQuiz/ArtQuizResults/ArtQuizResultsTabs/ArtQuizExploreArtworks.tsx (hunk 2 above)
Comment (rajsam003) [reply src/app/Scenes/ArtQuiz/ArtQuizResults/ArtQuizResultsTabs/ArtQuizExploreArtworks.tsx:30]: I followed what was on [force](https://github.com/artsy/force/blob/a0b7c135e8f37ee6007dad3f5a3a26869365b7de/src/Apps/ArtQuiz/Components/ArtQuizResultsRecommendedArtworks.tsx#L23). 
There we are doing the same thing. 
Should I leave it or change it?

Comment (TMcMeans) [reply src/app/Scenes/ArtQuiz/ArtQuizResults/ArtQuizResultsTabs/ArtQuizExploreArtworks.tsx:30]: Yep, there was a mistake made somewhere down the line in force where we also forgot to use this. Caught this yesterday and wrote a ticket here to fix: https://artsyproduct.atlassian.net/jira/software/c/projects/GRO/boards/96?modal=detail&selectedIssue=GRO-1572

Comment (TMcMeans) [reply src/app/Scenes/ArtQuiz/ArtQuizResults/ArtQuizResultsTabs/ArtQuizExploreArtworks.tsx:30]: Gonna hopefully have a PR up for that one by EOD today!

Comment (TMcMeans) [reply src/app/Scenes/ArtQuiz/ArtQuizResults/ArtQuizResultsTabs/ArtQuizExploreArtworks.tsx:30]: got that fix PR here: https://github.com/artsy/force/pull/11969#event-8584166844

**Review by MounirDhahri — APPROVED** [review]
Thanks for making these changes @rajsam003 👍
LGTM

**Comment by TMcMeans** [comment]
awesome work!

**Pull request merged**